### PR TITLE
Add Atlas/PMD

### LINKED ISSUE #1554: new module: atlas/PMD
I think it would be good to have a module for [atlas/PMD](https://bioconda.github.io/recipes/atlas/README.html)

- [x] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [x] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [x] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [ ] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by merszym**
# Atlas/PMD

This merge adds the module atlas/pmd to nf-core/modules

## PR checklist

Closes #1554  

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [X] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/atlas/pmd/main.nf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::atlas=0.9.9" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas:0.9.9--h082e891_0':
+        'quay.io/biocontainers/atlas:0.9.9--h082e891_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(fasta), path(fai), path(pool_rg_txt)
```
Comment (jfy133) [inline modules/atlas/pmd/main.nf:11]:
```suggestion
    tuple val(meta), path(bam), path(bai), path(fasta), path(fai), path(pool_rg_txt)
```

FASTA and FAI should be in separate channels

File: modules/atlas/pmd/meta.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+name: "atlas_pmd"
+description: Estimate the post-mortem damage patterns of DNA
+keywords:
+  - ancient
+  - DNA
```
Comment (jfy133) [inline modules/atlas/pmd/meta.yml:5]:
```suggestion
  - ancient DNA
```

File: modules/atlas/pmd/meta.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      description: The reference genome provided as FASTA file
+      pattern: "*.fasta"
+  - fai:
+      type: file
+      description: The FAI file for the reference genome FASTA file
+      pattern: "*.fai"
+  - pool_rg_txt:
+      type: file
+      description: |
+        (optional)! Provide the names of read groups that should be merged for PMD estimation.
```
Comment (jfy133) [inline modules/atlas/pmd/meta.yml:44]:
```suggestion
        Optional. Provide the names of read groups that should be merged for PMD estimation.
```

**Review by jfy133 — DISMISSED** [review]
Couple of meta tweaks, but fasta/fai should be in separte channels! (Don't forget to update the test too). Look at my PR for atlas/call

**Review by jfy133 — APPROVED** [review]
Free to merge @merszym !

**Comment by merszym** [comment]
> Free to merge @merszym !

Thanks for the review :)

**Pull request merged**